Makefile

```diff
@@ -268,6 +268,16 @@ BINARYNAME=$(TARNAME)-$(PLATFORM)-$(ARCH)
 BINARYTAR=$(BINARYNAME).tar.gz
 PKG=out/$(TARNAME).pkg
 PACKAGEMAKER ?= /Developer/Applications/Utilities/PackageMaker.app/Contents/MacOS/PackageMaker
+LEDATATAR=$(TARNAME)-locales-le.tar
+BEDATATAR=$(TARNAME)-locales-be.tar
+LEDATATGZ=$(LEDATATAR).gz
+BEDATATGZ=$(BEDATATAR).gz
+#ICU_DATA_IN=$(firstword $(wildcard deps/icu/source/data/in/icudt*l.dat))
```

Comment:
I would remove all comments from this PR if possible, otherwise it makes it difficult for reviewers to know if commented out code should be reviewed or not.